### PR TITLE
[IMM32][NTUSER] Implement ImmGetHotKey

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

JIRA issue: [CORE-11700](https://jira.reactos.org/browse/CORE-11700)

- Implement `imm32!ImmGetHotKey` function.
- Modify `NtUserGetImeHotKey` function.
- Add `LPHKL` typedef into `<imm.h>` header.

### CODE REVIEW

File: media/doc/README.WINE [hunk 1]
```diff
@@ -78,7 +78,7 @@ dll/win32/iccvid              # Synced to WineStaging-4.0
 dll/win32/ieframe             # Synced to WineStaging-4.18
 dll/win32/imaadp32.acm        # Synced to WineStaging-4.0
 dll/win32/imagehlp            # Synced to WineStaging-4.18
-dll/win32/imm32               # Synced to WineStaging-4.18
+dll/win32/imm32               # Forked at WineStaging-4.18
```
Comment (oleg-dubinskiy) [inline media/doc/README.WINE:81]: Place the implementation in `#ifdef __REACTOS__` instead. Why fork the whole component only for the one small implementation? It makes sense only if there are too many differences with Wine in the whole file.

Comment (katahiromz) [reply media/doc/README.WINE]: OK.

**Comment by katahiromz** [comment]
How can we call kernel function `NtUserGetImeHotKey` from imm32?

**Comment by RepOfCATV95** [comment]
It seems you got compile errors that need to be fixed ASAP.

**Comment by katahiromz** [comment]
Build is fixed,

**Comment by katahiromz** [comment]
```txt
c:/rosbe/i386/bin/../lib/gcc/i686-w64-mingw32/8.4.0/../../../../i686-w64-mingw32/bin/ld.exe: cannot find -llibwin32ksys_delayed
```

File: dll/win32/imm32/imm.c [hunk 2]
```diff
@@ -3160,11 +3160,27 @@ BOOL WINAPI ImmEnumInputContext(DWORD idThread, IMCENUMPROC lpfn, LPARAM lParam)
  *              ImmGetHotKey(IMM32.@)
  */
 
+#ifdef __REACTOS__
+BOOL APIENTRY NtUserGetImeHotKey(DWORD, LPUINT, LPUINT, LPHKL); /* FIXME */
```
Comment (learn-more) [inline dll/win32/imm32/imm.c:3164]: `win32ss/include/ntuser.h` has this (but a worse prototype)

Comment (katahiromz) [reply dll/win32/imm32/imm.c]: OK.

**Pull request merged**